### PR TITLE
Update LootLockerSDK settings

### PR DESCRIPTION
**Pull request by kirre-bylund**
* Deprecate Platform dropdown but leave it in for legacy
* Add tooltips and change naming to clarify api keys etc.
* Make development mode legacy and make it conditional on if there's a legacy api key.
* Deprecate platform property in config

Visual examples:

Deprecated platform: 
Depr platform

Depr platform fixed (or first install):
non depr platform

With a dev key:
devkey

With a prod key:
prodkey

With a legacy key:
legacykey

Added hover style tooltip for the API key:
tooltip

### CODE REVIEW

File: LootLockerSDK/Source/LootLockerSDK/Private/LootLockerPlatformManager.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+FLootLockerPlatformRepresentation& ULootLockerCurrentPlatform::CurrentPlatform = *PlatformRepresentations.Find(ELootLockerPlatform::None);
+
+const FString FLootLockerPlatformRepresentation::GetFriendlyStringFromEnum(const ELootLockerPlatform& Platform)
+{
+    return ULootLockerConfig::GetEnum(TEXT("ELootLockerPlatform"), static_cast<int>(Platform));
+}
+
+void ULootLockerCurrentPlatform::Set(const ELootLockerPlatformType& LegacyPlatform)
```
[the remark this answers is not in the document]
Comment (kirre-bylund) [reply LootLockerSDK/Source/LootLockerSDK/Private/LootLockerPlatformManager.cpp:25]: Interesting, let's investigate. The same should go for the change in the Unity SDK in that case

[the remark this answers is not in the document]
Comment (TobiasBerg) [reply LootLockerSDK/Source/LootLockerSDK/Private/LootLockerPlatformManager.cpp:25]: Android and guest are not the same in the backend, players are treated in different buckets, so we should support them both. 

Just for information though, the current Android platform will soon be deprecated, and a new Google Sign In will be added as well. 
We we still need to support the old platform, for the games already using it.

**Review by TobiasBerg — CHANGES_REQUESTED** [review]

File: LootLockerSDK/Source/LootLockerSDK/Private/LootLockerPlatformManager.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (kirre-bylund) [reply LootLockerSDK/Source/LootLockerSDK/Private/LootLockerPlatformManager.cpp:25]: ✅ I understood you the complete reverse when we had the prep chat 😂 I'll fix it (and in the Unity SDK as well)

[the remark this answers is not in the document]
Comment (kirre-bylund) [reply LootLockerSDK/Source/LootLockerSDK/Private/LootLockerPlatformManager.cpp:26]: Fixed now

**Review by TobiasBerg — APPROVED** [review]

**Pull request merged**